frontend/src/js/controllers/authCtrl.js

```diff
@@ -163,7 +164,7 @@
                                     vm.FormError = response.data.non_field_errors[0];
                                 }
                             } catch (error) {
-                                console.log(error);
+                                $rootScope.notify("error",error);
```

Comment:
Can you please replace this line with `$rootScope.notify(error);` ?

Reply:
Ready.